nautobot/dcim/templates/dcim/map_contact_or_team.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+
+{% block form %}
+    <div id="assign" class="tabcontent">
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Action</strong></div>
+            <div class="panel-body">
+                {% render_field form.action %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Location Data</strong></div>
+            <div class="panel-body">
+                {% render_field form.location %}
+                {% render_field form.name %}
+                {% render_field form.phone %}
+                {% render_field form.email %}
+                {% render_field form.address %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Contact Association</strong></div>
+            <div class="panel-body">
+                {% render_field form.contact %}
+                {% render_field form.team %}
+                {% render_field form.role %}
+                {% render_field form.status %}
+                {% render_field form.associated_object_type.as_hidden %}
+                {% render_field form.associated_object_id.as_hidden %}
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block buttons %}
+    <button type="submit" name="_create" class="btn btn-primary">Assign</button>
+    <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
+{% endblock %}
+
+{% block javascript %}
+<script>
+    document.getElementById("id_location").disabled=true;
+
+    // action drop down toggle
+    var action = document.getElementById("id_action");
```

Comment:
Is this similar to what we did in GC? https://github.com/nautobot/nautobot-app-golden-config/blob/develop/nautobot_golden_config/static/toggle_fields.js If so, can we follow that pattern (or perhaps a better one?)